helm-charts/konk-service/templates/configmap.yaml

```diff
@@ -13,7 +13,16 @@ data:
     CERT=$(cat certs/ca.crt | base64)
     CERT=${CERT//$'\n'/}
     cat /mounts/api-service.yaml.in | sed "s/{{` {{ SERVICENAME }} `}}/${SERVICENAME}/g" | sed "s/{{` {{ NAMESPACE }} `}}/${NAMESPACE}/g" | sed "s/{{` {{ CERT }} `}}/${CERT}/g" > /gen/api-service.yaml
-    kubectl apply -f /gen/api-service.yaml
+    delay=2
+    until kubectl apply -f /gen/api-service.yaml
+    do
+      sleep $delay
+      let delay="delay*2"
+      if [ "$delay" -gt "60" ]
+      then
+        delay=60
+      fi
+    done
```

Comment:
This retries adding the APIService to konk until konk is ready. Before adding this, it sometimes failed when racing to create the APIService before konk was ready.